examples/nl-normal-address.html

```diff
@@ -0,0 +1,132 @@
+<?php 
+/*  
+ *  Template for the MyParcel checkout.
+ *
+ */
+?>
+
+<script src="https://code.jquery.com/jquery-1.11.3.js"></script>
+<link rel="stylesheet" href="../css/myparcel.css">
+
+<div id="mypa-spinner"></div>
+<div id="mypa-message"></div>
+<div id="mypa-location-details"></div>
+
+<form id="mypa-delivery-option-form" class="" action="../" enctype="multipart/form-data" method="post" target="_self">
+	<div class="mypa-fieldset">
+		<div id="mypa-send-per-carrier">
+			<div id="mypa-txt-send-per-carrier">Bezorgopties</div>
+		</div>
+		<ul id="mypa-pickup-selector">
+			<li>
+				<input name="mypa-deliver-or-pickup" id="mypa-deliver-pickup-deliver" value="mypa-deliver" type="radio">
+				<label for="mypa-delivery-date">Bezorgen op</label>
+			</li>
+		</ul>
+		<ul id="mypa-delivery-selectors-nl">
+			<li>
+				<div class="full-width" id="mypa-delivery-date">
+					<select name="mypa-delivery-date" id="mypa-select-date">
+						<option value="">Geen datum</option>
+					</select>
+				</div>
+			</li>
+			<li class="mypa-delivery-option method-myparcel-delivery-morning-div">
+				<input name="shipping-method" id="method-myparcel-delivery-morning" type="radio" value="myparcel-morning">
+				<label for="method-myparcel-delivery-morning">Ochtendlevering <span id="mypa-morning-delivery"></span></label>
+			</li>
+			<li id="mypa-postnl-delivery" class="mypa-delivery-option method-myparcel-flatrate-div">
```

Comment:
Graag alles wat in de code staat als `flatrate`, maar eigenlijk `normal` delivery is, moet je even hernoemen van `flatrate` naar `normal`. `flatrate` is verwarrend en ik zou verwachten dat het iets is uit WC. Waarschijnlijk heeft Dennis deze term overgenomen uit WC.

Reply:
Done!